android/build.gradle

```diff
@@ -1,14 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    apply from: "dependencies.gradle"
+
     repositories {
         // this line will fix Could not find com.android.tools.build:gradle:3.0.1
         // https://stackoverflow.com/a/47456492/3135248
         google()
         jcenter()
+        google()
```

Comment:
Brilliant. This is not rebased at all... 😄

Reply:
I rebased it, but i didn't see this

Reply:
@vico1103 You didn't use the latest master so I have so many conflicts to resolve now. Basically, nothing changed from the last PR if I am being honest...

Reply:
@vico1103 Leave it, I'll try to merge it. I will let you know in case of any complications.

Reply:
I use master from 1 pm